service_mesh/v2x/upgrading-ossm-to-2-1.adoc

```diff
@@ -0,0 +1,94 @@
+[id="ossm-upgrade-to-2-1"]
+= Upgrading {ProductName} from version 2.0 to version 2.1
+include::modules/ossm-document-attributes.adoc[]
+:context: oossm-upgrading-to-2-1
+
+toc::[]
+
+Upgrading from version 2.0 to 2.1 requires manual steps that migrate your workloads and application to a new instance of {ProductName} running the new version.
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.7. before you upgrade to {ProductName} 2.0.
+* You must have {ProductName} version 2.1.0 operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. However, there are steps you must take to use the features in {ProductName} version 2.1.
```

Comment:
I'm confused about what to do here. The "However, there are steps you must take to use the features in {ProductName} version 2.1" sentence seems to repeat the introduction.
```suggestion
* You have the {ProductName} version 2.1.0 operator. If you selected the *automatic* upgrade path, the operator automatically downloads the latest information. Otherwise [get it from somewhere]. 
```